diff -rupN clsql-4.2.0/clsql.asd clsql-4.2.0-new/clsql.asd
--- clsql-4.2.0/clsql.asd	2009-12-11 13:57:46.000000000 -0700
+++ clsql-4.2.0-new/clsql.asd	2009-12-11 15:12:32.000000000 -0700
@@ -27,7 +27,7 @@
 ;; need to load uffi for below perform :after method
 (eval-when (:compile-toplevel :load-toplevel :execute)
   (unless (find-package 'uffi)
-    (asdf:operate 'asdf:load-op 'uffi)))
+    (asdf:operate 'asdf:load-op 'cffi-uffi-compat)))
 
 (defsystem clsql
     :name "CLSQL"
diff -rupN clsql-4.2.0/clsql-mysql.asd clsql-4.2.0-new/clsql-mysql.asd
--- clsql-4.2.0/clsql-mysql.asd	2009-09-02 06:52:24.000000000 -0700
+++ clsql-4.2.0-new/clsql-mysql.asd	2009-12-11 15:12:32.000000000 -0700
@@ -21,7 +21,7 @@
 
 (eval-when (:compile-toplevel :load-toplevel :execute)
   (unless (find-package 'uffi)
-    (asdf:operate 'asdf:load-op 'uffi)))
+    (asdf:operate 'asdf:load-op 'cffi-uffi-compat)))
 
 (defvar *library-file-dir* 
   (merge-pathnames "db-mysql/" 
@@ -80,7 +80,7 @@
   :description "Common Lisp SQL MySQL Driver"
   :long-description "cl-sql-mysql package provides a database driver to the MySQL database system."
 
-  :depends-on (uffi clsql clsql-uffi)
+  :depends-on (cffi-uffi-compat clsql clsql-uffi)
   :components
   ((:module :db-mysql
 	    :components
diff -rupN clsql-4.2.0/clsql-uffi.asd clsql-4.2.0-new/clsql-uffi.asd
--- clsql-4.2.0/clsql-uffi.asd	2009-07-27 14:20:37.000000000 -0700
+++ clsql-4.2.0-new/clsql-uffi.asd	2009-12-11 15:12:32.000000000 -0700
@@ -79,7 +79,7 @@
   :description "Common UFFI Helper functions for Common Lisp SQL Interface Library"
   :long-description "cl-sql-uffi package provides common helper functions using the UFFI for the CLSQL package."
 
-  :depends-on (uffi clsql)
+  :depends-on (cffi-uffi-compat clsql)
 
   :components
   ((:module :uffi
diff -rupN clsql-4.2.0/db-mysql/Makefile clsql-4.2.0-new/db-mysql/Makefile
--- clsql-4.2.0/db-mysql/Makefile	2009-11-16 20:55:52.000000000 -0700
+++ clsql-4.2.0-new/db-mysql/Makefile	2009-12-11 15:15:44.000000000 -0700
@@ -27,14 +27,14 @@ shared64_lib=$(base)64.so
 all: $(shared_lib)
 
 CFLAGS=-I /usr/local/include/mysql -I /usr/include/mysql -I /sw/include/mysql -I /opt/local/include/mysql -I /opt/local/include/mysql5/mysql -I /usr/local/mysql/include
-LDFLAGS=-L/usr/local/lib64/mysql -L/usr/local/mysql/lib32 -L/usr/local/lib/mysql -L/usr/lib/mysql -L/sw/lib -L/opt/local/lib/mysql -L/opt/local/lib/mysql5/mysql -L/usr/local/mysql/lib -L/usr/lib/gcc/i686-pc-cygwin/3.4.4 -lmysqlclient -lz -lc
+LDFLAGS=-L/usr/local/lib64/mysql -L/usr/local/mysql/lib32 -L/usr/local/lib/mysql -L/usr/lib64/mysql -L/usr/lib/mysql -L/sw/lib -L/opt/local/lib/mysql -L/opt/local/lib/mysql5/mysql -L/usr/local/mysql/lib -L/usr/lib/gcc/i686-pc-cygwin/3.4.4 -lmysqlclient -lz -lc
 
 CFLAGS32=-I /usr/local/include/mysql -I /usr/include/mysql -I /sw/include/mysql -I /opt/local/include/mysql -I /opt/local/include/mysql5/mysql -I /usr/local/mysql/include -m32
 LDFLAGS32=-L/lib32 -L/usr/lib32 -melf_i386 -lmysqlclient -lz -lc
 
 ifneq ($(OS_CYGWIN),0)
   CFLAGS=-I /cygdrive/c/Program\ Files/MySQL/MySQL\ Server\ 5.0/include
-  LDFLAGS=-L/usr/local/lib64/mysql -L/usr/local/lib/mysql -L/usr/lib/mysql -L/sw/lib -L/opt/local/lib/mysql -L/usr/lib/gcc/i686-pc-cygwin/3.4.4 -lmysqlclient -lpthread -lz -lm -lgcc -lc
+  LDFLAGS=-L/usr/local/lib64/mysql -L/usr/local/lib/mysql -L/usr/lib64/mysql -L/usr/lib/mysql -L/sw/lib -L/opt/local/lib/mysql -L/usr/lib/gcc/i686-pc-cygwin/3.4.4 -lmysqlclient -lpthread -lz -lm -lgcc -lc
 endif
 
 $(shared_lib): $(source) Makefile
